src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -1,51 +1,71 @@
 package blackjack.domain.player
 
 import blackjack.domain.card.Card
-import blackjack.domain.card.Card.AceCard
-import blackjack.domain.game.TakeMorePlayerStrategy
+import blackjack.domain.card.ReceivedCards
+import blackjack.domain.game.strategy.TakeMorePlayerStrategy
 
 open class Player(
     private val _name: String,
-    val takeMorePlayerStrategy: TakeMorePlayerStrategy,
-    val receivedCards: MutableSet<Card> = mutableSetOf()
+    private val _receivedCards: ReceivedCards = ReceivedCards(mutableSetOf()),
+    private val _gambleSummary: GambleSummary = GambleSummary()
 ) {
-
-    var isWinner: Boolean = false
+    constructor(name: String, cards: List<Card>) : this(_name = name) {
+        cards.map {
+            _receivedCards.addCard(it)
+        }
+    }
 
     val score: Int
-        get() = calculateScore()
+        get() = _receivedCards.calculateScore().score
 
     val name: String
         get() = _name
 
-    fun calculateScore(): Int {
-        var score = receivedCards.sumOf { it.number }
+    val receivedCards: ReceivedCards
+        get() = _receivedCards
 
-        if (score > BLACKJACK_SCORE) {
-            val aceCount = receivedCards
-                .count { it is AceCard }
+    val gambleSummary: GambleSummary
+        get() = _gambleSummary
 
-            score = score - (ACE_NUMBER_TO_ELEVEN * aceCount) + (ACE_NUMBER_TO_ONE * aceCount)
-        }
+    fun canMoreGame(): Boolean {
+        return _receivedCards.calculateScore().score < BLACKJACK_SCORE
+    }
 
-        return score
+    fun isBust(): Boolean {
+        return _receivedCards.calculateScore().score > BLACKJACK_SCORE
```

Comment:
```suggestion
        return _receivedCards.calculateScore().isBust()
```
원시값을 객체로 포장했을 때 잇점은 위임을 통해 결합도를 줄여 변화에 유연해지고 캡슐화를 할 수 있기 때문이에요.